pandas/core/generic.py

```diff
@@ -2199,14 +2200,15 @@ class  max_speed
         """
 
     @Appender(_shared_docs['take'])
-    def take(self, indices, axis=0, convert=True, is_copy=True, **kwargs):
-        nv.validate_take(tuple(), kwargs)
-
-        if not convert:
+    def take(self, indices, axis=0, convert=-1, is_copy=True, **kwargs):
```

Comment:
why can’t u just use None

Reply:
@jreback : breaks compatibility with older numpy versions

Reply:
not sure how